opentracing/event.go

```diff
@@ -0,0 +1,22 @@
+package opentracing
+
+import (
+	"time"
+
+	"sourcegraph.com/sourcegraph/appdash"
+)
+
+func init() {
+	appdash.RegisterEvent(spanCompletionEvent{})
```

Comment:
Sounds good to me. How about we rename this to `Timespan` and make it public within the `appdash` package?